### PR TITLE
docs(core): fix core stackblitz examples

### PR DESCRIPTION
**Pull request by Betrozov**
#### Please provide a link to the associated issue.
Closes SAP/fundamental-ngx#4391

#### Please provide a brief summary of this pull request.
Fixed StackBlitz examples these components:
 - date picker
- dialog
- datetime picker
- calendar
- file carousel
- file uploader
- formatted text
- standard list
- time
- table
- select
- popover
- pagination
- checkbox
- time picker
- toolbar
- message toast
- menu
- message box
- list with byline
- object status
- dynamic side content
- layout grid
- layout panel
#### Please check whether the PR fulfills the following requirements

- [X] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [X] tests for the changes that have been done
- [X] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [X] Run npm run build-pack-library and test in external application

Documentation checklist:
- [X] update `README.md`
- [X] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [X] Documentation Examples
- [X] Stackblitz works for all examples

### CODE REVIEW

**Review by InnaAtanasova — APPROVED** [review]

**Review by N1XUS — APPROVED** [review]

**Review by olshansky — APPROVED** [review]

**Comment by maxym-valor** [comment]
Nothing works. Error in each stackblitz.
image

File: apps/docs/src/app/core/component-docs/calendar/examples/calendar-options-example/calendar-options-example.component.ts [hunk 1]
```diff
@@ -1,8 +1,19 @@
 import { Component } from '@angular/core';
+import { DatetimeAdapter, DATE_TIME_FORMATS, FdDatetimeAdapter, FD_DATETIME_FORMATS } from '@fundamental-ngx/core';
```
Comment (Betrozov) [inline apps/docs/src/app/core/component-docs/calendar/examples/calendar-options-example/calendar-options-example.component.ts:2]:
```suggestion
import { DatetimeAdapter, DATE_TIME_FORMATS, FdDatetimeAdapter, FD_DATETIME_FORMATS } from '@fundamental-ngx/core/datetime';
```

File: apps/docs/src/app/core/component-docs/toolbar/examples/toolbar-example.component.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { Component } from '@angular/core';
+import { DatetimeAdapter, FdDatetimeAdapter } from '@fundamental-ngx/core';
```
Comment (Betrozov) [inline apps/docs/src/app/core/component-docs/toolbar/examples/toolbar-example.component.ts:2]:
```suggestion
import { DatetimeAdapter, FdDatetimeAdapter } from '@fundamental-ngx/core/datetime';
```

File: apps/docs/src/app/core/component-docs/toolbar/examples/toolbar-overflow-example.component.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { Component } from '@angular/core';
+import { DatetimeAdapter, DATE_TIME_FORMATS, FdDatetimeAdapter, FD_DATETIME_FORMATS } from '@fundamental-ngx/core';
```
Comment (Betrozov) [inline apps/docs/src/app/core/component-docs/toolbar/examples/toolbar-overflow-example.component.ts:2]:
```suggestion
import { DatetimeAdapter, DATE_TIME_FORMATS, FdDatetimeAdapter, FD_DATETIME_FORMATS } from '@fundamental-ngx/core/datetime';
```

File: apps/docs/src/app/core/component-docs/calendar/examples/calendar-grid-example/calendar-grid-example.component.ts [hunk 4]
```diff
@@ -1,10 +1,21 @@
 import { Component } from '@angular/core';
+import { DatetimeAdapter, DATE_TIME_FORMATS, FdDatetimeAdapter, FD_DATETIME_FORMATS } from '@fundamental-ngx/core';
```
Comment (N1XUS) [inline apps/docs/src/app/core/component-docs/calendar/examples/calendar-grid-example/calendar-grid-example.component.ts:2]: Please use deep import here

**Comment by web-dev23** [comment]
> Nothing works. Error in each stackblitz.
> image

Fixed.

**Comment by nikvalor** [comment]
Date Picker - Formatting

image

**Comment by nikvalor** [comment]
Datetime picker - Formatting
image

**Comment by maxym-valor** [comment]
Does not work all examples for Leyout Grid
image

**Review by aberikashvili — APPROVED** [review]
LGTM

**Comment by web-dev23** [comment]
> Date Picker - Formatting
> 
> image

Fixed.

**Comment by web-dev23** [comment]
> Does not work all examples for Leyout Grid
> image

Fixed.

**Comment by nikvalor** [comment]
> Date Picker - Formatting
> 
> image

Fixed!

**Comment by web-dev23** [comment]
@nikvalor , please wait until latest changes are deployed.

**Comment by maxym-valor** [comment]
does not workTable with Non-navigatable Row
image

**Comment by maxym-valor** [comment]
Does not work message box: complex template
image

**Comment by nikvalor** [comment]
> Does not work message box: complex template
> image

Fixed

**Comment by nikvalor** [comment]
> does not workTable with Non-navigatable Row
> image

Fixed

**Pull request merged**